packages/select-rich/src/LionSelectRich.js

```diff
@@ -369,6 +378,31 @@ export class LionSelectRich extends SlotMixin(ScopedElementsMixin(OverlayMixin(L
     this._overlayCtrl.removeEventListener('hide', this.__overlayOnHide);
   }
 
+  /**
+   * Align invoker width with content width
+   * Make sure display is not set to "none" while calculating the content width
+   */
+  _alignInvokerWidth() {
+    if (this._overlayCtrl && this._overlayCtrl.content) {
```

Comment:
Our internal extension doesn't need this behavior, since it uses a global overlay. Should i restrict this to local overlays?

Reply:
Hmmm, but in theory one could create a lion select rich extension using global DOM position but may still want the invoker width to be aligned. Perhaps leave it up to the subclasser to add a guard if they don't want the behaviour.